### ISSUE TITLE
Add slider to the project list 

### ISSUE THREAD

**Issue by anikdhabal** [layer5io/layer5#4464]
### Current Behavior

 Projects list are out of the page in general as well as in small screen.
### Desired Behavior

Make this a carousal. 
### Screenshots / Mockups

![Screenshot 2023-06-29 192927](https://github.com/layer5io/layer5/assets/81948346/b63a7cce-3f5c-441e-a83f-3c3e38c74543)

### Acceptance Tests

---
<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.webp" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

**Comment by RakshitKumar04** [comment]
Hi @anikdhabal cI would like to work on this issue could you please assign this issue to me?

**Comment by Ghat0tkach** [comment]
@RakshitKumar04 there You go , Dont forger to update your status as well as to ask any doubts regarding the issue on [slack](https://layer5io.slack.com/archives/C015QJKUMPU) channel

**Comment by RakshitKumar04** [comment]
Hey @anikdhabal , @Ghat0tkach can you please give more details about the issue so that I have a clearer idea?

**Comment by Ghat0tkach** [comment]
> Hey @anikdhabal , @Ghat0tkach can you please give more details about the issue so that I have a clearer idea?

Sure, as you see the companies name arent clearly visible (it is a feature not bug as cited by Lee but you're free to make it carousal)

Now what is carousal? Well i got you
If you go into layer5.io website , you'll find companies have a slider 
![Screenshot_20230701-235845-458.png](https://github.com/layer5io/layer5/assets/59855919/0d965a4d-9475-4494-b11a-3bd76131677e)

You have to incorporate the same.

**Comment by anikdhabal** [comment]
> > Hey @anikdhabal , @Ghat0tkach can you please give more details about the issue so that I have a clearer idea?
> 
> Sure, as you see the companies name arent clearly visible (it is a feature not bug as cited by Lee but you're free to make it carousal)
> 
> Now what is carousal? Well i got you If you go into layer5.io website , you'll find companies have a slider ![Screenshot_20230701-235845-458.png](https://user-images.githubusercontent.com/59855919/250299635-0d965a4d-9475-4494-b11a-3bd76131677e.png)
> 
> You have to incorporate the same.

Thanks @Ghat0tkach. Refer this @RakshitKumar04.

**Comment by RakshitKumar04** [comment]
Hii, @Ghat0tkach I have sent the PR, please take a look.

**Comment by ashutosh887** [comment]
@Ghat0tkach @RakshitKumar04 
If it's not resolved yet, I would like to work on this

**Comment by RakshitKumar04** [comment]
@ashutosh887 I am working on it.

**Comment by RakshitKumar04** [comment]
Yes, I am working on it. I was unable to continue my work, reason my laptop crashed. I will soon update the PR.

**Comment by Vayras** [comment]
is this issue still pending, I would like to contribute towards this

**Comment by Ghat0tkach** [comment]
@Vayras do you want to work on this issue?

**Comment by Vayras** [comment]
@Ghat0tkach if it is still pending I can work on this

**Comment by Ghat0tkach** [comment]
@Vayras , there you go

**Comment by Vayras** [comment]
I'm in a bit of a pickle and was wondering if you could lend me a hand. I've been trying to run my project on my local machine using the trusty old "npm run" and "npm start" commands. Everything seemed to be on track, but here's the catch: when I navigate to localhost, all I'm seeing is the GraphQL page instead of the actual site preview I was hoping for. Your expertise in helping me sort this out would mean the world to me. Thanks a bunch!

**Comment by saurabh100ni** [comment]
This issue has been open for some time with no recent activity, unassigning to open it up for new contributors to give it a go.

**Comment by Ashparshp** [comment]
@saurabh100ni Can you provide the page url?

**Comment by saurabh100ni** [comment]
@Ashparshp This issue is invalid now, so closing this. Thanks for the heads up.
